.github/actions/hugo-build-action/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: Hugo Build
```

Comment:
Please add new or extend existing README with how to use the action and how to do its operations on localhost if needed.

Reply:
Will be resolved in follow-up.